src/resources/LibraryStyles.css

```diff
@@ -1,780 +1,808 @@
-    padding: 5px 0px 5px 0px;
+  padding: 5px 0px 5px 0px;
 }
 
 .SearchResultItemContainer .ItemIcon,
 .SearchResultItemContainerSelected .ItemIcon {
-    padding: 2px 8px;
-    min-width: 32px;
-    width: 32px;
-    min-height: 32px;
-    height: 32px;
+  padding: 2px 8px;
+  min-width: 32px;
+  width: 32px;
+  min-height: 32px;
+  height: 32px;
 }
 
 .SearchResultItemContainer .ItemTitle,
 .SearchResultItemContainerSelected .ItemTitle {
-    margin-bottom: 2px;
-    font-size: 1.2rem;
+  margin-bottom: 2px;
+  font-size: 1.2rem;
 }
 
 .SearchResultItemContainer .ItemDescription,
 .SearchResultItemContainerSelected .ItemDescription {
-    font-size: 1rem;
-    padding: 2px 0px;
-    color: #aaa;
+  font-size: 1rem;
+  padding: 2px 0px;
+  color: #aaa;
 }
 
 .SearchResultItemContainer .ItemDetails,
 .SearchResultItemContainerSelected .ItemDetails {
-    display: flex;
-    align-items: center;
-    font-size: 1rem;
-    color: #aaaaaa;
+  display: flex;
+  align-items: center;
+  font-size: 1rem;
+  color: #aaaaaa;
 }
 
 .SearchResultItemContainer .ItemParent,
 .SearchResultItemContainerSelected .ItemParent {
-    display: inline-block;
-    padding-right: 5px;
-    transition: 0.15s;
-    text-decoration: underline;
+  display: inline-block;
+  padding-right: 5px;
+  transition: 0.15s;
+  text-decoration: underline;
 }
 
 .SearchResultItemContainer .ItemParent:hover,
 .SearchResultItemContainerSelected .ItemParent:hover {
-    cursor: pointer;
-    color: white;
+  cursor: pointer;
+  color: white;
 }
 
 .SearchResultItemContainer .ItemCategory,
 .SearchResultItemContainerSelected .ItemCategory {
-    display: inline-block;
-    color: #ddd;
-    padding-left: 5px;
+  display: inline-block;
+  color: #ddd;
+  padding-left: 5px;
 }
 
 .SearchResultItemContainer .ItemTypeIcon,
 .SearchResultItemContainerSelected .ItemTypeIcon {
-    width: 1rem;
-    height: 1rem;
-    margin-top: auto;
-    margin-bottom: auto;
+  width: 1rem;
+  height: 1rem;
+  margin-top: auto;
+  margin-bottom: auto;
 }
 
 .HighlightedText {
-    font-weight: 700;
-    color: #4ac8ef;
+  font-weight: 700;
+  color: #4ac8ef;
 }
 
 .SearchResultItemContainer .ItemParent:hover .HighlightedText {
-    color: white;
+  color: white;
 }
 
 ::-webkit-input-placeholder {
-    color: #aaaaaa;
+  color: #aaaaaa;
 }
 
 ::-webkit-scrollbar {
-    width: 6px;
-    height: 6px;
-    background-color: #414141;
+  width: 6px;
+  height: 6px;
+  background-color: #414141;
 }
 
 ::-webkit-scrollbar-thumb {
-    width: 6px;
-    border-radius: 3px;
-    background-color: rgba(136, 136, 136, 0.8);
+  width: 6px;
+  border-radius: 3px;
+  background-color: rgba(136, 136, 136, 0.8);
 }
 
 ::-webkit-scrollbar-corner {
-    background-color: inherit;
+  background-color: inherit;
+}
+
+.tooltipWrapper {
+  height: fit-content;
+}
+
+.ClusterLeftPane .customTooltip {
+  background: #d9d9d9;
+  color: #2a2a2a;
+  padding: 5px;
+  font-family: "Artifakt Element", "Open Sans";
```

Comment:
The sonar-cube is pointing that we have not added a generic font to the font family. sans-serif is the generic option. That should fix it.

Reply:
@reddyashish I've added sans serif as generic font but SonarCloud still fails :/

Reply:
Passing now without the quotes to family font.